src/Image.ts

```diff
@@ -941,6 +993,20 @@ export class Image {
   ): Image {
     return correctColor(this, measuredColors, referenceColors);
   }
+  /**
+   * Apply a median filter to the image.
+   *
+   * @param options - Options to apply for median filter.
+   * @param options.radius - Size of the area to calculate median value from.
```

Comment:
You should only keep options, all the other parameters are defined in MedianFilterOptions